install/installer/pkg/components/blobserve/configmap.go

```diff
@@ -101,7 +101,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 				MaxSize:  MaxSizeBytes,
 			},
 		},
-		AuthCfg:            "/mnt/pull-secret.json",
+		AuthCfg:            "/mnt/pull-secret/pull-secret.json",
```

Comment:
I forget, but, nesting the secret within a sub-directory was necessary, so that we can watch it for changes, right?

Reply:
You're right 💯

Reply:
FYR https://kubernetes.io/docs/concepts/configuration/secret/#:~:text=A%20container%20using%20a%20Secret%20as%20a%20subPath%20volume%20mount%20does%20not%20receive%20automated%20Secret%20updates